src/main.rs

```diff
@@ -33,7 +33,7 @@ mod tests;
 
 lazy_static! {
     // NOTE: Use [[:alnum::]] instead of \w to only match ASCII word characters, not unicode
-    static ref MANGLED_NAME_PATTERN: Regex = Regex::new(r"_ZN[\$\._[:alnum:]]*").unwrap();
+    static ref MANGLED_NAME_PATTERN: Regex = Regex::new(r"_[RZ]N[\$\._[:alnum:]]*").unwrap();
```

Comment:
`_R` symbols don't necessarily start with `_RN`.

Reply:
I see, you're right -- for reference: https://github.com/rust-lang/rfcs/blob/master/text/2603-symbol-name-mangling-v2.md#syntax-of-mangled-names